src/chainstate/coordinator/mod.rs

```diff
@@ -2742,7 +2742,7 @@ impl<
                     // Was this block sufficiently confirmed by the prepare phase that it was a PoX
                     // anchor block?  And if we're in epoch 2.1, does it match the heaviest-confirmed
                     // block-commit in the burnchain DB, and is it affirmed by the majority of the
-                    // network?
+                    // network? TODO(sbtc): What are the additional checks we need in 3.0 here?
```

Comment:
Let's get this closed before merging.

It could be that no changes are needed.  Altering the validation rules for block-commits might be sufficient to enforce all of the new forking rules.

Reply:
Makes sense. I plan to break out the Epoch 3.0 changes and the peg-in wire format to two separate PR:s, giving us better separation between the changes and allowing us to run an alpha version of the Peg-in change before introducing Epoch 3.0.